calcom/cal.com
# feat: domain wide delegation UI

## What does this PR do?



- Fixes #XXXX (GitHub issue number)
- Fixes CAL-XXXX (Linear issue number - should be visible at the bottom of the GitHub issue description)

First PR for domain wide delegation

- UI for adding domain wide delegation and workspace platform
- Team Feature flags check

Splitting a large PR https://github.com/calcom/cal.com/pull/16622 into smaller ones




https://github.com/user-attachments/assets/4cd96973-e063-44c9-a4ff-f5d3d171a331






![Screenshot 2024-12-05 at 4 32 30 PM](https://github.com/user-attachments/assets/bc2bd551-f83a-47cf-972b-9574d881f180)




## Mandatory Tasks (DO NOT REMOVE)


## How should this be tested?

1) Make sure you are logged in as admin user and Enable domain wide delegation  in settings/admin/flags 
2) Go to settings/admin/workspace-platforms to add workspace platform and edit workspace platform
3) Go to settings/organizations/domain-wide-delegation to test adding and removing dwd

Also create a team feature entry to enable it
